packages/eslint-config/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@wordpress/eslint-config",
+	"version": "1.0.0-alpha.0",
+	"description": "Eslint config for WordPress development.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"eslint"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/eslint-config/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"dependencies": {
+		"babel-eslint": "^8.0.3",
+		"eslint-config-wordpress": "2.0.0",
```

Comment:
As discussed in Core/JS chat, Let's copy this dependency here and ship.

Reply:
Done with e612065.